### PR TITLE
Add libftdi

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge/help-c-cpp this is ready for review
Windows is currently skipped because of https://github.com/conda-forge/libusb-feedstock/issues/9 - I am happy to add that once it's fixed.

/cc @wolfv - we need ftdi for a few of the remaining ROS packages

File: recipes/libftdi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  host:
+    - libusb
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/libftdi1/ftdi.h  # [unix]
+    - test -f ${PREFIX}/lib/libftdi1${SHLIB_EXT}  # [osx]
+    - test -f ${PREFIX}/lib/pkgconfig/libftdi1.pc  # [osx]
+    - test -f ${PREFIX}/lib/cmake/libftdi1/LibFTDI1Config.cmake  # [osx]
+    - test -f ${PREFIX}/lib64/libftdi1${SHLIB_EXT}  # [linux]
```
Comment (wolfv) [inline recipes/libftdi/meta.yaml:33]: We should  use "-DCMAKE_INSTALL_LIBDIR=lib" to install into lib, not lib64. 

https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html

**Review by wolfv — CHANGES_REQUESTED** [review]
Looks great, small comment

File: recipes/libftdi/meta.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply recipes/libftdi/meta.yaml:33]: Hmm, I already pass this argument: https://github.com/conda-forge/staged-recipes/blob/be6929f34060f3aa06d9fbb6c9d9b81bf3debb5b/recipes/libftdi/build.sh#L11

Comment (wolfv) [reply recipes/libftdi/meta.yaml:33]: https://github.com/lipro/libftdi/blob/9da4394efeafa2fcfdc809a9bec3219c0a95e77a/CMakeLists.txt#L52-L61

Looks like we need `-DLIB_SUFFIX=""`

**Comment by Tobias-Fischer** [comment]
Thanks @wolfv!

**Pull request merged**